ev3dev/helper.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import logging
+import sys
+import time
+import ev3dev.auto
+from ev3dev.auto import OUTPUTS, RemoteControl, list_motors
+from math import pi
+from time import sleep
+
+log = logging.getLogger(__name__)
+
+
+def wait_for(condition, timeout=1e5, interval=0.01):
+    tic = time.time() + timeout
+    done = condition()
+
+    while time.time() < tic and not done:
+        time.sleep(interval)
+        done = condition()
+
+    return done
+
+
+# =============
+# Motor classes
+# =============
+class MotorMixin(object):
+
+    def wait_for_running(self):
+        return wait_for(lambda: 'running' in self.state, 1)
+
+    def wait_for_stop(self):
+        return wait_for(lambda: 'running' not in self.state, 1)
+
```

Comment:
Does this eliminate the need in [wait functions](https://github.com/rhempel/ev3dev-lang-python/pull/118/files#diff-52ffd7ac54a3727e62910423b8b5a485R621) in the core module? Should those be removed from there?